C/kmsg_log.c

```diff
@@ -14,14 +14,14 @@
 int klog(char *msg)
 {
 
- char *buf = malloc(sizeof(char) * 256);
+ char *buf = malloc(256);
```

Comment:
ok: https://stackoverflow.com/questions/419022/char-x256-vs-char-malloc256sizeofchar/419029#419029
btw char buf[256]; could be better.